pkg/cmd/diagnostics/diagnostics_download.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func newDiagnosticsDownloadCommand(ctx api.Context) *cobra.Command {
+func newDiagnosticsDownloadCommand(_ api.Context) *cobra.Command {
```

Comment:
Why not remove this parameter completely when we're not using it?

Reply:
Fixed